Common.md

```diff
@@ -20,3 +20,130 @@ Language names are title cased.
+
+The following table is an overview for quick reference all the attributes defined in the following sub-sections.
+
+| Attribute | REQUIRED/RECOMMENDED/OPTIONAL |
+| --------- | ----------------------------- |
+| `instrumentation.provider` | REQUIRED |
+| `instrumentation.name` | RECOMMENDED |
+| `instrumentation.version` | RECOMMENDED |
+| `collector.name` | RECOMMENDED |
+| `collector.version` | RECOMMENDED |
+| `host.hostname` | REQUIRED† |
+| `service.name` | REQUIRED† |
+| `app.name` | REQUIRED† |
+| `db.name` | REQUIRED† |
+
+†: REQUIRED only if annotating an applicable telemetry source.
+See the related sub-section for more information.
+
+### Instrumentation
+
+Instrumentation is the toolset used to measure telemetry.
+This instrumentation usually comes in the form of a library of code, platform, or coding language builtin and provides different instruments to measure the state of a computer program.
+This instrumentation is from what the exporter exports.
+
+Including attributes to identify not only what instrumentation is used but also what code is being instrumented is crucial for interoperability between exporters and the New Relic platform.
+
+#### `instrumentation.provider`
+
+All exported telemetry MUST contain an `instrumentation.provider` attribute that identifies the provider of instrumentation.
+
+Example values of this attribute are:
+
+* `newRelic`
+* `prometheus`
+* `dropwizard`
+* `opencensus`
+* `opentelemetry`
+
+#### `instrumentation.name`
+
+All exported telemetry SHOULD contain an `instrumentation.name` attribute that identifies the code that is being instrumented.
+This would be the framework, library, or application that uses the instrumentation to report telemetry.
+
+Example values of this attribute are:
+
+* `spring-framework`
+* `python-twisted`
+* `gorilla/mux`
+
+#### `instrumentation.version`
+
+All exported telemetry SHOULD contain an `instrumentation.version` attribute that identifies the version of the instrumentation.
+This will be defined by the instrumentation provider.
+
+Example values of this attribute are:
+
+* `v0.1.0`
+* `3`
+* `alpha-v1`
+
+### Collector
+
+Telemetry sent to New Relic is often produced by many collocated sources.
+Given the close proximity these telemetry sources can have, it can be advantageous to unify the transmission of this telemetry to New Relic.
+This reduces network overhead and can simplify the design of system architectures.
+
+Being able to backtrack identity what collector was the ultimate transmitter of telemetry to New Relic is often useful.
+It can help debug issues and provide context of data flows.
+The following attributes are used to make this identification possible.
+
+#### `collector.name`
+
+All exported telemetry SHOULD contain an `collector.name` attribute that identifies the name of the code that collected the telemetry.
+This value SHOULD be overwritten if telemetry is proxied.
```

Comment:
what does "telemetry is proxied" mean in this context?

Reply:
I was intending it to mean it was sent through an intermediary before being routed to New Relic. But, the understanding I have of what this attribute means come from you. If you have a better way to phrase I'm open to suggestions.

Reply:
like, if went through the OTel collector, for example?

Reply:
I'd add a small section explain telemetry proxies to inform this and the bits below that reference the term.